docs/_layouts/default.html

```diff
@@ -57,6 +57,7 @@ <h2>Setup</h2>
                     <li><a href="#bundling">Bundling</a></li>
                     <li><a href="#flags">Flags</a></li>
                     <li><a href="#cdn">CDN</a></li>
+                    <li><a href="#legacy-rounded-off">Legacy Rounded Off</a></li>
```

Comment:
Might end up renaming this before we do final release, but it's good for now. Perhaps a "Property Packs" or "Theme Packs" section or the like...